modules/features/profile/src/main/java/au/com/shiftyjelly/pocketcasts/profile/ProfileFragment.kt

```diff
@@ -169,8 +172,11 @@ class ProfileFragment : BaseFragment() {
 
         viewLifecycleOwner.lifecycleScope.launch {
             viewLifecycleOwner.repeatOnLifecycle(Lifecycle.State.STARTED) {
-                val isEligible = viewModel.isEndOfYearStoriesEligible()
-                binding.setupEndOfYearPromptCard(isEligible)
+                if (viewModel.shouldDisplayKidsProfileBanner()) {
+                    binding.setupKidsCard()
+                } else if (viewModel.isEndOfYearStoriesEligible()) {
+                    binding.setupEndOfYearPromptCard()
+                }
```

Comment:
Should End of Year card be not displayed if the Kids Profile card is displayed?

Reply:
I confirmed with Davi that should not be displayed at the same time, but he suggested to display kids first in case we want to display both of them at the same time